### PR TITLE
#125 Use version catalogs for ktlint plugin

### LINKED ISSUE #125: Setup ktlint as precompiled plugin

### PR DESCRIPTION
**Pull request by MarekMacko**
Closes #125

### CODE REVIEW

File: buildSrc/build.gradle.kts [hunk 1]
```diff
@@ -5,3 +5,7 @@ plugins {
 repositories {
 	mavenCentral()
 }
+
+dependencies {
+	implementation(files(libs.javaClass.superclass.protectionDomain.codeSource.location))
```
Comment (BorysChajdas) [inline buildSrc/build.gradle.kts:10]: What is that for?

File: buildSrc/settings.gradle.kts [hunk 2]
```diff
@@ -0,0 +1,7 @@
+dependencyResolutionManagement {
```
Comment (BorysChajdas) [inline buildSrc/settings.gradle.kts:1]: I suppose it's needed because `libs.versions.toml` is not used by default in this module?

Comment (MarekMacko) [reply buildSrc/settings.gradle.kts:1]: Right

File: buildSrc/build.gradle.kts (hunk 1 above)
Comment (MarekMacko) [reply buildSrc/build.gradle.kts:10]: To access `LibrariesForLibs`

**Review by BorysChajdas — APPROVED** [review]

**Review by marcinWisnia — APPROVED** [review]

**Pull request merged**